File: python/main1.py
import random

from Quicksort import Quicksort

arr = []

for _ in range(10):
    arr.append(int(random.random() * 100))

print(f'Antes: {arr}')

sort = Quicksort()
sort.quicksort(arr, 0, len(arr)-1)

print(f'Despues: {arr}')

File: java/Main1.java
import java.util.Random;

public class Main1 {
    public static void main(String[] args) {
        Random rand = new Random();

        int[] arr = new int[25];

        for (int i=0; i<arr.length; i++){
            arr[i] = rand.nextInt(100);
        }

        System.out.print("Before: ");
        printArray(arr);

        Quicksort sort = new Quicksort();
        sort.quicksort(arr);

        System.out.print("After: ");
        printArray(arr);       
    }

    private static void printArray(int[] arr){
        for (int num : arr) {
            System.out.print(num + " ");
        }
        System.out.println("");
    }
}